src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -198,6 +200,9 @@ internal class CommandLineParameterParser
     {
         private const int MaxPipePathLengthLinux = 108;
         private const int MaxPipePathLengthMacOS = 104;
+        internal static int MaxNameLength() => Platform.IsWindows ? ushort.MaxValue : (Platform.IsLinux ? MaxPipePathLengthLinux : MaxPipePathLengthMacOS) - Path.GetTempPath().Length;
```

Comment:
Please make this an instance method, also format the body to make it readable.

Reply:
Reformated.